dynabuffers-java/src/test/kotlin/dynabuffers/usecase/Schema26Test.kt

```diff
@@ -0,0 +1,34 @@
+package dynabuffers.usecase
+
+import dynabuffers.AbstractDynabuffersTest
+import dynabuffers.Dynabuffers
+import dynabuffers.DynabuffersEngine
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+
+class Schema26Test : AbstractDynabuffersTest() {
+
+    lateinit var engine: DynabuffersEngine;
+
+    @BeforeEach
+    fun setUp() {
+        engine = Dynabuffers.parse(Schema08Test::class.java.getResourceAsStream("/schema26.dbs"))
```

Comment:
Statt `Schema08Test` könntest du `this` verwenden

Reply:
Das passiert durch die blöde umakopiererrei von den SchemaX Tests :-)
Ich stell die aber bald alle um, also lass ich das jetzt einfach noch so.